build/android-build.py

```diff
@@ -38,21 +38,24 @@ def caculate_built_samples(args):
     targets = set(targets)
     return list(targets)
 
-def do_build(app_android_root, build_mode):
+def do_build(app_android_root, build_mode, app_abi):
 
-    command = 'cocos compile -p android -s %s --ndk-mode %s' % (app_android_root, build_mode)
+    command = 'cocos compile -p android -s %s --ndk-mode %s --app-abi %s' % (app_android_root, build_mode, app_abi)
     print command
 
     if os.system(command) != 0:
         raise Exception("Build dynamic library for project [ " + app_android_root + " ] fails!")
 
-def build_samples(target, build_mode):
+def build_samples(target, build_mode, app_abi):
 
     if build_mode is None:
         build_mode = 'debug'
     elif build_mode != 'release':
         build_mode = 'debug'
 
+    if app_abi is None:
+        app_abi = 'armeabi-v7a'
```

Comment:
Why use `armeabi-v7a` by default, it will break compatibility. I think we should discuss it again in forum before changing it. I [discussed it](http://discuss.cocos2d-x.org/t/how-about-use-armeabi-v7a-as-default-abi-on-android/17247) last time, and many developers don't agree with it.